scriptworker/log.py

```diff
@@ -88,7 +88,8 @@ def get_log_filenames(context):
     Returns:
         tuple: log file path, error log file path
     """
-    log_file = os.path.join(context.config['task_log_dir'], 'task_output.log')
+    # XXX Even though our logs aren't live, Treeherder looks for live_backing.log to show errors in failures summary
+    log_file = os.path.join(context.config['task_log_dir'], 'live_backing.log')
```

Comment:
I think we should use `task_error.log` as the `live_backing.log`.
Currently `task_error.log` contains `STDERR` + `STDOUT`; `task_output.log` has `STDOUT`.  If we use `task_output.log`, we'll drop everything that goes to `STDERR`.

Also, in practice, I haven't found `task_output.log` useful.  We could actually move to one combined task log file.  If you're up for it, let's do it.  If you want to limit scope, we can stay with `live_backing.log` + a mostly useless `task_output.log`.